src/components/common/header/header.css

```diff
@@ -1,117 +1,137 @@
 /*--------head------------*/
 .head {
-  padding: 20px 0;
-  color: #fff;
+	padding: 20px 0;
+	color: #fff;
 }
 
 .logo h1 {
-  font-size: 35px;
-  font-weight: 900;
+	font-size: 35px;
+	font-weight: 900;
 }
 
 .head .icon {
-  margin-left: 10px;
+	margin-left: 10px;
 }
 
 .flexSB {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
 }
 
 .button {
-  font-size: 2rem;
-  font-weight: bolder;
+	font-size: 2rem;
+	font-weight: bolder;
 }
 
 /*--------head------------*/
 /*--------header------------*/
 header {
-  background-color: #6c63ff;
-  margin: 30px 30px;
-  padding: 0 20px;
+	/* margin: 30px auto; */
+	padding: 20px 0px;
+	display: flex;
+	justify-content: center;
+	position: fixed;
+	z-index: 1;
+	width: 100%;
+	transition: 0.2s ease-in-out;
 }
 
-header .nav {
-  display: flex !important;
+/* .header-color {
+	background-color: #6c63ff;
+} */
+
+header.scrolled {
+	background-color: #6c63ff;
+	padding: 0px 0px;
+}
```

Comment:
We want the colored navigation as it was previously

It should be having that margins before getting scrolled and after scrolling it should be gone and work as it is working now